src/test/java/de/kherud/llama/LlamaModelIT.java

```diff
@@ -157,14 +171,13 @@ public void testCompleteGrammar() {
 	@Test
 	public void testEmbedding() {
 		float[] embedding = model.embed(prefix);
-		Assert.assertEquals(5120, embedding.length);
```

Comment:
Different dimensions of model between codellama and mistral 7b, so the embeddings have different dims.